EKF/ekf_helper.cpp

```diff
@@ -746,12 +727,14 @@ void Ekf::getAuxVelInnovRatio(float &aux_vel_innov_ratio) const
 
 void Ekf::getFlowInnov(float flow_innov[2]) const
 {
-	memcpy(flow_innov, _flow_innov, sizeof(_flow_innov));
+	flow_innov[0] = _flow_innov(0);
```

Comment:
You can do these (and the variance, and the mag/mavVar) with `_flow_innov.copyTo(flow_innov);`